static/home/css/style.css

```diff
@@ -234,6 +246,9 @@ nav{
 .navbar2 li a{
 	display: block;
 }
+.navbar2 a:hover{
```

Comment:
Don't make these changes, for now, we are handling navbar2 in a different issue.